src/sentry/models/dashboard_permissions.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import Model, region_silo_model
+from sentry.db.models.base import sane_repr
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@region_silo_model
+class DashboardPermissions(Model):
+    """
+    Edit permissions for a Dashboard.
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+
+    created_by_id = HybridCloudForeignKey("sentry.User", on_delete="CASCADE")
```

Comment:
I don't think we need to store the `created_by_id` on this model, that would be available metadata in the dashboard.